src/main/java/com/dropbox/core/DbxSdkVersion.java

```diff
@@ -32,7 +33,7 @@ private static String loadLineFromResource()
     {
         try {
             InputStream in = DbxSdkVersion.class.getResourceAsStream(ResourceName);
-            if (in == null) throw new LoadException("Not found.");
+            if (in == null) return "3.0.10"; // <-- needs to instead be auto generated and inserted into code by the build system // throw new LoadException(" Not found.");
```

Comment:
I think it's better to report "unknown" here.

My worry is that it's easy to forget to update this value.  In that case, it's probably better to report "unknown" than report the wrong version.  (Either way, this will only happen in the rare case that "sdk-version.txt" isn't present.)

Reply:
On the other hand, if you're sure you will update the version every time, then there's no need for the "sdk-version.txt" file anymore.